backend/ircconnection/app.ts

```diff
@@ -60,7 +62,7 @@ rl.on('line', (line) => {
     console.log(`TRYING TO JOIN ${joinConfig.channel}`);
   } else if (ircMessage.command == 'JOIN') {
     const nick = ircMessage.prefix && ircMessage.prefix.split('!')[0].slice(1);
-    if (nick != 'tstestbot') myEmitter.emit('join', ircMessage.params[0].split(' ', 1)[0].replace(':', ''), nick);
+    if (nick != 'lmrdashboard') myEmitter.emit('join', ircMessage.params[0].split(' ', 1)[0].replace(':', ''), nick);
```

Comment:
Can you put that into the joinConfig and use it similarly to `joinConfig.channel`?